src/tests/src/libs/antares/antlr4-interface/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+find_package(antlr4-runtime CONFIG REQUIRED)
+Set(SRCS test_antlr_interface.cpp
+)
+
+set(execname "antlr-interface-test")
+add_executable(${execname}  ${SRCS})
+target_link_libraries(${execname}
+                       PRIVATE
+                       antlr-interface
+		               Boost::unit_test_framework
+                     )  
+
+
+target_include_directories(${execname} 
+        PRIVATE
+         ${ANTLR4_INCLUDE_DIR})
```

Comment:
If it's needed by users of the lib, we need to make it public in the CMakeList of the lib itself.
Even better would be to be able to add it to the antlr4-runtime imported target, but I don't know if it's possible.